.github/workflows/test-application.yaml

```
@@ -28,3 +28,166 @@ jobs:
+                    MYSQL_ROOT_PASSWORD: root
+                ports:
+                    - 3306:3306
+                options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5
+
+            jackrabbit:
+                image: sulu/jackrabbit:2.18
+                env:
+                    DATABASE_HOST: mysql
+                    DATABASE_PORT: 3306
+                    DATABASE_NAME: sulu_test_jackrabbit
+                    DATABASE_USER: root
+                    DATABASE_PASS: root
+                    LOG_LEVEL: WARN
+                ports:
+                    - 8080:8080
+
+            elasticsearch5:
+                image: elasticsearch:5.6
+                ports:
+                    - 9201:9200
+                env:
+                    discovery.type: 'single-node'
+                    xpack.security.enabled: 'false'
+                options: --health-cmd="curl http://localhost:9200/_cluster/health" --health-interval=10s --health-timeout=5s --health-retries=5
+
+            elasticsearch7:
+                image: elasticsearch:7.9.3
+                ports:
+                    - 9202:9200
+                env:
+                    discovery.type: 'single-node'
+                    xpack.security.enabled: 'false'
+                options: --health-cmd="curl http://localhost:9200/_cluster/health" --health-interval=10s --health-timeout=5s --health-retries=5
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: ${{ matrix.php-extensions }}
+                  tools: ${{ matrix.tools }}
+                  coverage: xdebug
+
+            - name: Get composer cache directory
+              id: composer-cache-dir
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              id: composer-cache
+              with:
+                  path: ${{ steps.composer-cache-dir.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-composer-
+
+            - name: Require elasticsearch dependency
+              run: composer require --dev elasticsearch/elasticsearch:"^${{ matrix.elasticsearch-version }}" --no-interaction --no-update
+
+            - name: Require phpstan dependency
+              if: ${{ matrix.phpstan }}
+              run: composer require --dev phpstan/phpstan --no-interaction --no-update
+
+            - name: Install dependencies
+              run: |
+                  composer validate
+                  composer update ${{ matrix.composer-flags }}
+
+            - name: Override parameters.yml file
```

is this still needed ? 🤔  as far as i see, there is no `Tests/app/config/parameters.yml` file in the repo. but i dont know how these files work tbh

---

there is a `Tests/app/config/parameters.yml.dist` which will be copied as `Tests/app/config/parameters.yml` after `composer install`

---

alright - maybe adjust the name then?
```suggestion
            - name: Clear parameters.yml file to use environment variables
```